<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>12</y>
     <width>37</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*메일:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_mails">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>10</y>
     <width>225</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>44</y>
     <width>37</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*이름:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_names">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>42</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>76</y>
     <width>37</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*전화:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_telnos">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>74</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_addrs">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>104</y>
     <width>227</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>106</y>
     <width>37</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>*주소:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sns">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>132</y>
     <width>227</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>134</y>
     <width>37</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SNS:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_edit">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>184</y>
     <width>89</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>수정</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>184</y>
     <width>89</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>160</y>
     <width>229</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>* 는 필수 입력사항 입니다.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
